.github/workflows/tests.yml

```diff
@@ -0,0 +1,156 @@
+      # Re-build packages, ensuring packages/webdriverjs/dist exists.
+      - run: npx lerna run prepare
+      # lerna link after installing the latest version of chromedriver, as we depend on webdriverJS
+      - run: npx lerna link
+        working-directory: packages/cli
+      - run: npm run coverage --prefix=packages/cli
+
+  webdriverjs:
+    strategy:
+      matrix:
+        node: [16, 18, 20]
+    runs-on: ubuntu-latest
+    needs: lint
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node }}
+          cache: 'npm'
+      # HACK: Force a TTY to enable browser-driver-manager to manipulate stdout.
+      - shell: 'script -q -e -c "bash {0}"'
+        run: npx browser-driver-manager install chrome chromedriver --verbose
+      - run: npm run build --prefix=packages/webdriverjs
+      - run: npm run coverage --prefix=packages/webdriverjs
+      - run: npm run test:esm --prefix=packages/webdriverjs
+
+  webdriverio:
+    strategy:
+      matrix:
+        node: [16, 18, 20]
+    runs-on: ubuntu-latest
+    needs: lint
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node }}
+          cache: 'npm'
+      # HACK: Force a TTY to enable browser-driver-manager to manipulate stdout.
+      - shell: 'script -q -e -c "bash {0}"'
+        run: npx browser-driver-manager install chrome chromedriver --verbose
+      - run: npm run build --prefix=packages/webdriverio
+      - run: npm run coverage --prefix=packages/webdriverio
+      - run: npm run test:esm --prefix=packages/webdriverio
+
+  reporter_earl:
+    strategy:
+      matrix:
+        node: [16, 18, 20]
+    runs-on: ubuntu-latest
+    needs: lint
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node }}
+          cache: 'npm'
+      - run: npm run build --prefix=packages/reporter-earl
+      - run: npm run test --prefix=packages/reporter-earl
+      - run: npm run test:esm --prefix=packages/reporter-earl
+
+  react:
+    strategy:
+      matrix:
+        node: [16, 18, 20]
+    runs-on: ubuntu-latest
+    needs: lint
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node }}
+          cache: 'npm'
+      - run: npm run build --prefix=packages/react
+      - run: npm run test --prefix=packages/react
+      - run: npm run test:esm --prefix=packages/react
+
+  playwright:
+    strategy:
+      matrix:
+        node: [16, 18, 20]
+    runs-on: ubuntu-latest
+    needs: lint
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node }}
+          cache: 'npm'
+      - run: npx playwright install --with-deps
```

Comment:
If this job is a long-pole, we might consider running it on a playwright docker image pinned to the same version as our playwright version instead of `ubuntu-latest` - they come with playwright deps + matching browser versions pre-installed

Reply:
Good idea! This job does take significantly longer than similar jobs.

<img width="271" alt="playwright vs puppeteer" src="https://github.com/dequelabs/axe-core-npm/assets/571265/7044aa51-6397-4e1b-912b-e6150559cd7d">

Are you OK with leaving this as-is and using the Playwright Docker image in a followup PR? I can likely get to it within the next few days.